engine/flink/executor/src/main/scala/pl/touk/nussknacker/engine/process/compiler/MetricsProviderForFlink.scala

```diff
@@ -0,0 +1,19 @@
+package pl.touk.nussknacker.engine.process.compiler
+
+import org.apache.flink.api.common.functions.RuntimeContext
+import pl.touk.nussknacker.engine.api.process.ComponentUseCase
+import pl.touk.nussknacker.engine.util.metrics.{MetricsProviderForScenario, NoOpMetricsProviderForScenario}
+
+object MetricsProviderForFlink {
+
+  def getMetricsProvider(
```

Comment:
Let's rename it to `createMetricProvider`. `get` looks like sth is already created and you only return it.